alerting/pagerduty.go

```diff
@@ -29,33 +28,45 @@ type PagerdutyEvent struct {
 	Details      map[string]interface{} `json:"details"`
 }
 
+// NewPagerdutyClient is an initilisation helper
 func NewPagerdutyClient(key string) *PagerdutyClient {
 	return &PagerdutyClient{integrationKey: key}
 }
 
-func (p PagerdutyClient) TriggerAlert(service services.Service, alert Alert, details map[string]interface{}) error {
+// AlerterCreate is a helper function to wrap CreateEvent
+func (p *PagerdutyClient) AlerterCreate(serviceName string, logger zap.Logger, alert *Alert, details map[string]interface{}) error {
+	return p.CreateEvent(serviceName, logger, alert, "trigger", details)
+}
+
+// AlerterResolve is a helper function to wrap CreateEvent
+func (p *PagerdutyClient) AlerterResolve(serviceName string, logger zap.Logger, alert *Alert, details map[string]interface{}) error {
+	return p.CreateEvent(serviceName, logger, alert, "resolve", details)
+}
+
+// CreateEvent is a generic func to create pagerduty events
+func (p *PagerdutyClient) CreateEvent(serviceName string, logger zap.Logger, alert *Alert, action string, details map[string]interface{}) error {
```

Comment:
same here - logger does not look like a parameter of an event creation - it could be  a state of a PagerDutyClient or even a global variable of a package